Add `Expert Global Taiko Showdown 2025: Registrations Now Open!` news post (#12633)
ppy/osu-wiki: news/2025/2025-01-05-expert-global-taiko-showdown-2025-registrations-now-open.md

---
layout: post

> title: "Expert Global Taiko Showdown 2025: Registrations now open!"

Reviewer: title case for titles

```suggestion
title: "Expert Global Taiko Showdown 2025: Registrations Now Open!"
```

Regarding the tournament's format, two changes are to be noted: instead of 96 players making it to a double-elimination bracket like in 2022, only the top **64 players** following a qualifier round will square off for glory. Mappool-wise, all songs used in the tournament will either come **from the Featured Artist listing**, or are **used with permission from the artist**.

The tournament's detailed schedule and all relevant info for participating can be found below.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | Jan 05 (00:00 UTC) – Jan 19 (23:59 UTC) |
| Screening phase | Jan 19 (00:00 UTC) – Feb 02 (23:59 UTC) |
| Qualifier showcase | Feb 02 (15:00 UTC) |
| Qualifier stage | Feb 08 & Feb 09 |
| Round of 64 | Feb 15 & Feb 16 |
| Round of 32 | Feb 22 & Feb 23 |
| Round of 16 | Mar 01 & Mar 02 |
| Quarterfinals | Mar 08 & Mar 09 |
| Semifinals | Mar 15 & Mar 16 |
| Finals | Mar 22 & Mar 23 |
| All Stars & Grand Finals | Mar 29 & Mar 30 |

## Original content

> As we've stated earlier before, EGTS releases a lot of original content through a variety of mediums, but the two main ones we have been known for have easily been for one, our story, which has been shown through extracts before reveals of all past EGTS Tiebreaker original songs, as well as the COEGTS 2023 Mappool Showcase.

Reviewer: two is mentioned but there's no second one apart from the story thing

```suggestion
As we've stated earlier, EGTS releases a lot of original content through a variety of mediums. One the main ones, however, has been our story, which has been shown through short videos before the reveals of all past EGTS tiebreaker original songs, as well as the [COEGTS 2023 mappool showcase](https://www.youtube.com/watch?v=0I3UH7Al-lM).
```
